contrib/masterfiles-stage/README.org

```diff
@@ -191,35 +200,99 @@ and the presence of an ssh key for authentication. Depending on the
 security requirements of your organization, you may use a
 passphraseless key, or you may use something like =ssh-agent=.
 
-
 * Example Usage
 :PROPERTIES:
 :ID:       67b5ead8-5bdf-47d9-a26a-70ad95b742d5
 :END:
 
 The script can be invoked manually:
 
-#+BEGIN_EXAMPLE
-  /var/cfengine/bin/masterfiles-stage.sh -d /tmp/deployed_masterfiles -p /var/cfengine/masterfiles-stage_parmas/PARAMS_example_git_branch.sh
-  tail -n1 /var/cfengine/outputs/dc-scripts.log
-  Successfully deployed 'master' from 'https://gitlab.com/nickanderson/masterfiles_demo_3.7.git' to '/var/cfengine/masterfiles' on Sun Dec  6 23:00:34 UTC 2015
-#+END_EXAMPLE
+#+CAPTION: Example manually executing masterfiles-stage
+#+begin_example
+  [root@hub ~]# ${DC_SCRIPTS}/masterfiles-stage.sh --DEBUG
+  Option Deploy Dir: /var/cfengine/masterfiles
+  Option Params File: /opt/cfengine/dc-scripts/params.sh
+  Cloning into bare repository '/opt/cfengine/https___gitlab.com_nickanderson_masterfiles-vcs'...
+  warning: redirecting to https://gitlab.com/nickanderson/masterfiles-vcs.git/
+  remote: Enumerating objects: 135, done.
+  remote: Counting objects: 100% (135/135), done.
+  remote: Compressing objects: 100% (116/116), done.
+  remote: Total 135 (delta 18), reused 123 (delta 13), pack-reused 0
+  Receiving objects: 100% (135/135), 201.34 KiB | 1.94 MiB/s, done.
+  Resolving deltas: 100% (18/18), done.
+  Successfully deployed '3.15.1' from 'https://gitlab.com/nickanderson/masterfiles-vcs' to '/var/cfengine/masterfiles' on Wed Apr 15 19:52:54 UTC 2020
+#+end_example
 
 Or execution can be automated with CFEngine.
 
+To enable automatic deployment define =cfengine_internal_masterfiles_update= and ensure that =cfe_internal_update_from_repository= is run as part of the update policy. Note, =cfe_internal_update_from_repository= is run automatically when =cfengine_internal_masterfiles_update= is defined AND CFEngine Enterprise binaries are in use using the stock update policy.
```

Comment:
s/Note, /Note that / maybe. I'm not sure if "Note, ..." is proper.

Reply:
I dunno, I prefer `Note,`. The pause makes it, more notable.